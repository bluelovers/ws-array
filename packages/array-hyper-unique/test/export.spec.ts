import _m0 = require('../');
import _m1 from '../';
import _m2 = require('../core');

test(`import check`, () =>
{

	let actual = Object.keys(_m0).sort();
	let expected = Object.keys(_m1).sort();
	let expected2 = Object.keys(_m2).sort();

	expect(actual).toStrictEqual(expected);
	expect(actual).toStrictEqual(expected2);
	//expect(actual).toBeInstanceOf(Date);
	expect(actual).toMatchSnapshot();

});

test(`__esModule`, () =>
{

	expect(_m0).toHaveProperty('__esModule', true);
	expect(_m1).toHaveProperty('__esModule', true);

});
