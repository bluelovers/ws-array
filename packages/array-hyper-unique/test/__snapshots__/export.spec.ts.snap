// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import check 1`] = `
Array [
  "array_unique",
  "array_unique_overwrite",
  "lazy_unique_overwrite",
  "equals",
  "defaultFilter",
  "defaultChecker",
  "lazy_unique",
  "default",
]
`;
